code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -91,9 +91,6 @@
 
 /mob/living/simple_animal/drone/Login()
 	..()
-	update_inv_hands()
-	update_inv_head()
-	update_inv_internal_storage()
```

Comment:
this was only here to update hud stuff, no longer needed since handled in living/Login()